### DISCUSSION TITLE
SARAL decoding

### DISCUSSION THREAD

**Discussion by N6RFM** [daniestevez/gr-satellites#344] [General]
Hi Dani and gr-satellites users!  I'm trying to decode SARAL NORAD 39086.  The telemetry format is listed as BPSK PMT-A3, 800 baud.  I've created a satyaml file with the following parameters, but have not had any success.

name: SARAL
alternative_names:
  - CVJK-7644-2588-4230-2659
norad: 39086
data:
  &tlm Telemetry:
    telemetry: ax25
transmitters:
   800k BPSK downlink:
    frequency: 465.988e+6
    modulation: BPSK
    baudrate: 800
    framing: AX.25 G3RUH
    data:
    - *tlm
   800k BPSK downlink:
    frequency: 465.988e+6
    modulation: BPSK
    baudrate: 800
    framing: AX.25
    data:
    - *tlm

A 120mb IQ file may be found here.  50,000 sps.  https://mega.nz/file/CtlV1Kyb#XJnTLxaSphUGbJ2_KTTpGb9FUSERukqKcEicvhPYFsc
Any suggestions or advice would be appreciated.

Thanks!
Bob N6RFM

**Comment by daniestevez** [comment]
Hi Bob,

From a quick look this 465.988 MHz transmitter is something used to relay data from ocean buoys. This PMT-A3 protocol is not something currently supported by gr-satellites, and a custom deframer for it would be needed.

**Reply by N6RFM** [reply]
Thanks Dani.  Indeed an esoteric format.  Some are claiming "decodes" using a Satnogs decoder.  I'll try to get some specifics.  In any case, not a driver for a new deframer .  Best, Bob
